### ISSUE TITLE
Prisma Launcher won't open.

### ISSUE THREAD

**Issue by Sudden-Snowball** [PrismLauncher/PrismLauncher#2885]
### Operating System

Windows

### Version of Prism Launcher

latest

### Version of Qt

I don't know

### Description of bug

Hi! I have a problem: the launcher does not start. I rebooted the computer, reinstalled the Prisma Launcher — nothing helps. Everything was working fine a week ago, and now here it is... I don't know what to do. Where can I find logs and all that? I deleted the metacache file that I found using the Everything program, but it didn't help.

### Steps to reproduce

1: Double click prismlauncher.exe
2: Nothing.

### This issue is unique

- [X] I have searched the issue tracker and did not find an issue describing my bug.

**Comment by Trial97** [comment]
vcredist is required to run Prism on Windows
Like most apps on Windows, you have to install vcredist for Prism to run. Depending on what version of Prism you are using, you may need a different version.

You need:

    [vcredist 2022 x64](https://aka.ms/vs/17/release/vc_redist.x64.exe) if you're using PrismLauncher-Windows-MSVC (the recommended version for Windows 10 64 bit/Windows 11).
    [vcredist 2022 arm64](https://aka.ms/vs/17/release/vc_redist.arm64.exe) if you're using PrismLauncher-Windows-MSVC-arm64 (the recommended version for Windows 10/11 on ARM).


See the [wiki page](https://prismlauncher.org/wiki/overview/frequent-issues/#%22msvcp140_2.dll-was-not-found%22) on Prism's website for more information.


Basically make sure you have the latest vscred(like reinstall it using the latest link)

**Comment by Sudden-Snowball** [comment]
![image](https://github.com/user-attachments/assets/4e17cf96-f5fc-4fc3-b158-9bd70becb68d)
This is installed.

**Comment by Sudden-Snowball** [comment]
Hey, thanks for the help. Reinstalling the vc_redist fixed the problem.
